caikit_computer_vision/modules/segmentation/models.py

```diff
@@ -84,24 +84,18 @@ def load(cls, model_file: Union[str, "ModuleConfig"]) -> "ViTSegmenter":
     def save(
         self,
         model_path: str,
-        segmentation_dirname: str = "image_segmentation",
     ):
         """Save the in-memory model to the given path.
 
         Args:
             model_path: str
                 Path that we want to export the segmentation model to.
-            segmentation_dirname: str
-                Subdirectory to which we want to save the segmentation model.
-                Default: image_segmentation
         """
         saver = ModuleSaver(
             self,
             model_path=model_path,
         )
-        segmentation_rel_path, segmentation_abs_path = saver.add_dir(
-            segmentation_dirname
-        )
+        segmentation_rel_path, segmentation_abs_path = saver.add_dir(model_path)
```

Comment:
just a note - `add_dir` is going to create a nested directory inside of `model_path`. E.g., if you call `save()` with `model_path=foo`, it's going to put the artifacts into `foo`.

It should work because it's being saved into the config, and pulled back out at load time, but it would probably make more sense to call this directory `segmentation_model` or something so that if you're inspecting the model artifacts, you know what to look for

Reply:
Do I understand you suggestion correctly? so we use e.g. "segmentation_model" as relative path like: `saver.add_dir("segmentation_model", model_path)`